bucket/islc.json

```
@@ -0,0 +1,24 @@
+{
+    "homepage": "https://www.wagnardsoft.com/",
+    "description": "Utility that will monitor and clear the memory standby list when it is > 1000mb (1gb). It may help users who have stutters in games when using windows 10 Creator update and higher version of windows.",
+    "license": "Freeware",
+    "version": "1.0.1.1",
+    "url": "https://www.wagnardsoft.com/ISLC/ISLC%20v1.0.1.1.exe#/ISLC-1.0.1.1.7z",
+    "hash": "73bd954fc0f786455cad60dc73d44fd4cd6230763f034f2e7aa3709db06e5952",
+    "extract_dir": "ISLC v1.0.1.1",
+    "shortcuts": [
+        [
+            "Intelligent standby list cleaner ISLC.exe",
+            "Intelligent Standby List Cleaner (ISLC)"
+        ]
+    ],
+    "persist": "Intelligent standby list cleaner ISLC.exe.Config",
+    "checkver": {
+        "url": "https://www.wagnardsoft.com/",
+        "re": "Intelligent standby list cleaner [Vv]([\\d\\.]*) Released"
+    },
+    "autoupdate": {
+        "url": "https://www.wagnardsoft.com/ISLC/ISLC%20v$version.exe#/ISLC-$version.7z",
```

Do not rename with versoin variable. It is redundant and do not help it anything.

```suggestion
        "url": "https://www.wagnardsoft.com/ISLC/ISLC%20v$version.exe#/ISLC.7z",
```